### DISCUSSION TITLE
Is it possible to implement rest image counter overlay in the last image?

### DISCUSSION THREAD

**Discussion by shakilAhmed50** [igordanchenko/react-photo-album#170] [Q&A] [answered]
Lets say i have total 20 images. But initially i want to keep 10 images visible and in the last image there will be an overlay indicating remaining number of images. I have attached an image as an example. Can we do this is react photo album?

![gallery_layout](https://github.com/igordanchenko/react-photo-album/assets/159689244/31bcd7b1-192d-4eb2-971c-2768406ec6fc)

**Comment by igordanchenko** [comment, marked as answer]
You can definitely implement this feature with `renderPhoto` function. Please take a look at the example [here](https://github.com/igordanchenko/react-photo-album/issues/74#issuecomment-1186336288).
